canonical/ubuntu.com
# Remove more unused Python dependencies and upgrade deps

## Done

This is another PR to split this PR: https://github.com/canonical/ubuntu.com/pull/12389 in smaller chunks

- More python depencies, a few of them depend on the store so @jpmartinspt could you check if they didn't break anything
- Removed pybreaker, since we are not using it anymore
- Didn't upgrade mistune since it requires this issue to be fixed first: https://github.com/canonical/canonicalwebteam.templatefinder/pull/25

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/
    - QAQAQAQAQAQAQAQA
